pkg/config/loader_test.go

```diff
@@ -473,7 +473,7 @@ sdsServer:
 				"KUMA_API_SERVER_AUTH_ALLOW_FROM_LOCALHOST":                                                "false",
 				"KUMA_MONITORING_ASSIGNMENT_SERVER_GRPC_PORT":                                              "3333",
 				"KUMA_MONITORING_ASSIGNMENT_SERVER_PORT":                                                   "2222",
-				"KUMA_MONITORING_ASSIGNMENT_SERVER_FETCH_TIMEOUT":                                          "45s",
+				"KUMA_MONITORING_ASSIGNMENT_SERVER_DEFAULT_FETCH_TIMEOUT":                                  "45s",
```

Comment:
We will need a line in Upgrade.md for this name change.

Reply:
Let me check, but I don't think this has been included in a release yet. Would we still need one if that is the case?

Reply:
Nope, we won't need it if this is a brand new config.

Reply:
Confirmed this is not in 1.1.x: 
https://github.com/kumahq/kuma/blob/be98608a8344d3480ae9f3e84d1d2a9a629793ad/pkg/config/mads/config.go#L19-L26